codebasin/schema/compilation-database.schema

```diff
@@ -24,17 +24,17 @@
       },
       "output": {
         "type": "string"
-      },
+      }
```

Comment:
Just checking - I did put this into a validator and it checks out, but I wanted to make sure removing the commas are okay because JSON is weirdly finicky about this kind of thing?

Reply:
Removing the commas is _necessary_ because JSON is weirdly finicky. :smile:

Before this change, the old `.schema` file was never actually passed into `jsonschema`.  The schema that we passed into `jsonschema` was constructed from a Python object -- there were extra commas in the Python object definition, but converting that object to a JSON string and back sorted it out.  It wasn't until I tried to actually use the contents of this `.schema` file directly that I discovered it had extra (invalid) commas.